src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(fromFileName);
+        writeTofile(report, toFileName);
+    }
+
+    private void writeTofile(String report, String toFileName) {
+        File file = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
+        }
+    }
 
+    private String createReport(String fromFileName) {
+        File file = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                String[] valueContent = value.split(",");
+                if (valueContent[0].equals("supply")) {
+                    supply += Integer.parseInt(valueContent[1]);
+                } else if (valueContent[0].equals("buy")) {
```

Comment:
do we need `else if` or `else` will be enough?

Reply:
I left if else, because if we will change code in future there may be file with another operation type.

Reply:
if in future there will be file with another operation type - we will need to refactor the code rather than write new `else if`

let's remove it for now

Reply:
ok